src/containers/upcoming-events/ducks/types.ts

```diff
@@ -28,4 +28,5 @@ export interface EventProps {
     end: Date;
   };
   price: number;
+  ticketCount?: number;
```

Comment:
This is interesting. Part of me feels like we are conflating two data structures here: 
- Event information for the general viewer
- Event information in the context of someone signed up for the event

I think it's fine the way this is now, but if we want to be pedantic, we should have something like:
`interface EventInformation`
`interface EventRegistration extend EventInformation`